src/DataWrangling/ECCO/ECCO_restoring.jl

```diff
@@ -20,43 +21,41 @@ import Oceananigans.OutputReaders: new_backend, update_field_time_series!
 struct ECCONetCDFBackend{N} <: AbstractInMemoryBackend{Int}
     start :: Int
     length :: Int
+    maxiter :: Int
```

Comment:
should we make it more clear this refers to inpainting?

Reply:
I have changed it to `inpainting_maxiter`. I am not super happy about it though. Do you have a suggestion for a name for this variable?

Reply:
How about making a new struct, `Inpainting`, which has a property `maxiter` (and which may be generalized to include more settings in the future)